src/MAUI/Maui.Samples/Samples/Scene/ShowLabelsOnLayer3D/ShowLabelsOnLayer3D.xaml.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Data;
+using Esri.ArcGISRuntime.Geometry;
+using Esri.ArcGISRuntime.Mapping;
+using Esri.ArcGISRuntime.Symbology;
+using Esri.ArcGISRuntime.Tasks;
+using Esri.ArcGISRuntime.Tasks.Offline;
+using Esri.ArcGISRuntime.UI;
+using Esri.ArcGISRuntime.ArcGISServices;
+using Esri.ArcGISRuntime.UI.Controls;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Esri.ArcGISRuntime.Mapping.Labeling;
+using Esri.ArcGISRuntime.Portal;
+
+namespace ArcGIS.Samples.ShowLabelsOnLayer3D
+{
+    [ArcGIS.Samples.Shared.Attributes.Sample(
+        "Show labels on layer in 3D",
+        "Scene",
+        "add later",
+        "")]
+    [ArcGIS.Samples.Shared.Attributes.OfflineData()]
+    public partial class ShowLabelsOnLayer3D
+    {
+        // Hold the ID of the portal item, which is a web scene.
+        private const string ItemId = "850dfee7d30f4d9da0ebca34a533c169";
+
+        public ShowLabelsOnLayer3D()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Try to load the default portal, which will be ArcGIS Online.
+                ArcGISPortal portal = await ArcGISPortal.CreateAsync();
+
+                // Create the portal item.
+                PortalItem websceneItem = await PortalItem.CreateAsync(portal, ItemId);
+
+                // Create the scene.
+                MySceneView.Scene = new Scene(websceneItem);
```

Comment:
IMO this sample could be simplified to just creating a Scene from an item URL, e.g.

```cs
        private const string ItemUrl = "https://www.arcgis.com/home/item.html?id=850dfee7d30f4d9da0ebca34a533c169";
// ...
        MySceneView.Scene = new Scene(new Uri(ItemUrl));
```